webcompat/templates/issue-wizard-form.html

```diff
@@ -337,7 +337,7 @@ <h3 class="page-heading">Filing a web compatibility issue</h3>
                     type={{ 'submit' if anonymous_reporting else 'button' }}
                     {{ '' if anonymous_reporting else 'disabled' }}
                     >
-                    {{'Report Anonymously' if anonymous_reporting else 'Anonymous reports temporarily disabled'}}
+                    {{'Report anonymously' if anonymous_reporting else 'Anonymous reports temporarily disabled'}}
             </button>
           </div>
```

Comment:
(lowercase change is coming from https://github.com/webcompat/webcompat.com/issues/3006)